content/docs/en/elements/components/list-picker.md

```diff
@@ -1,5 +1,25 @@
 ---
 title: ListPicker
+apiRef: https://docs.nativescript.org/api-reference/classes/_ui_list_picker_.listpicker
 ---
 
-TODO
+The Listpicker component allows you to pick a value from a list.
```

Comment:
Should refer to "the user" instead of "you" (you refers to the developer, while ´the user´ refers to the end user)

Reply:
Will keep this in mind for future references